portfolio/src/main/webapp/style.css

```diff
@@ -1,5 +1,5 @@
 body{
-    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-family: Arial, Helvetica, "Helvetica Neue", sans-serif;
```

Comment:
Why is this font in quotes specifically?

Reply:
It contains a space in the name. So quotes bind the name as one

Reply:
Thanks for clarifying.